_docs/management/continuous-deployment.md

```diff
@@ -150,3 +150,58 @@ Replace `DEPLOYER_SERVICE_ACCOUNT_USERNAME`, `ORG`, and `SPACE` with your inform
 You can also review their [sample circle.yml file](https://circleci.com/docs/1.0/config-sample/) for more configuration options.
 
 **Note**: If your `manifest.yml` describes more than one app, you might want to specify which app to push in the `cf push` line.
+
+### GitHub Actions
+
+See the [GitHub Actions documentation](https://help.github.com/en/actions) to get started, and this pre-made [GitHub Action](https://github.com/cloud-gov/cg-cli-tools).
+
+#### Usage
+
+After following the instructions for setting up a [cloud.gov service account](https://cloud.gov/docs/services/cloud-gov-service-account/), store you username (CG_USERNAME) and password (CG_PASSWORD) as [encrypted secrets](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets). 
+
+#### Sample workflow
+
+The following is an example of a workflow that uses this action. This example shows how to deploy a simple [].NET Core app](https://github.com/cloud-gov/cf-hello-worlds/tree/master/dotnet-core) to cloud.gov
+
+```yml
+name: .NET Core Deploy
```

Comment:
I like the use of .NET Core Deploy as an example, because it demonstrates using a language outside of what we ourselves normally work with that is still well-supported, and in the case of .NET, noting the fact that it's `.NET Core`.  Thank you!

Reply:
👍